### ISSUE TITLE
Question about only train CNN_affinity for crossdock_default2018.

### ISSUE THREAD

**Issue by RJ-Li** [gnina/models#29]
Hello developers!

I saw the format of types for training both CNN_score and CNN_affinity needs rmsd and affinity label, but I don't wanna train or use CNN_score in my work, so I am searching for how to make it only for CNN_affinity.

But different papers have different types file format, such as in `data/PDBBind2016/Refined_types`:
<label> <pK> <RMSD to crystal> <Receptor filename> <Ligand filename> # <Autodock Vina score>>
    but it is different in `data/refined` like:
```
0 -6.3979 10gs/10gs_rec.gninatypes 10gs/10gs_ligand_0.gninatypes # 5.31559 -8.06592
0 -6.3979 10gs/10gs_rec.gninatypes 10gs/10gs_ligand_1.gninatypes # 9.14515 -8.0171
```

I want to know how to write my own types file format if I only want to train CNN_affinity? 
AND what should I do when using training.py if I only train the model `--cnn crossdock_default2018`? It's new to me to use Caffe, so I don't know what kind of file should I use, and where weights_file should be assigned in.

(I have all ligand's rmsd so if removing CNN_score is very difficult I can accept it.)

**Comment by dkoes** [comment]
You can adjust the model file (https://github.com/gnina/models/blob/master/acs2018/default2018.model) to only use the labels you want in the loss function.

**Comment by RJ-Li** [comment]
Hi, just ask you to make sure the built-in `crossdock_default2018.caffemodel` is trained on `crossdock2020 dataset` or `PDBbind_v2019_refined-set`? 
I only know its redocking power is tested on PDBbind_v2019_refined-set, and `crossdock2020` is generated based on  Pocketome.

**Comment by dkoes** [comment]
The crossdock prefix means it was trained on the crossdocked set.

**Comment by Dadiao-shuai** [comment]
Have you trained the **default2018.model** on **PDBbind_v2020/v2019**? I read the article `Three-Dimensional Convolutional Neural Networks and a CrossDocked Data Set for Structure-Based Drug Design` , which says you trained them on PDBbind_v2016
